features/tags_and_wrangling/tag_set.feature

```diff
@@ -55,6 +55,16 @@ Feature: creating and editing tag sets
     And I should see "Knots Landing"
     And I should see "Models Inc"
 
+  Scenario: If a tag set does not exist, no one should be able to see its associations
+  Given I am logged in as "tagsetter"
+  When I view associations for a tag set that does not exist
+  Then I should see an error flash alert saying "What tag set did you want to look at?"
```

Comment:
It's probably better to keep things simple and do just `Then I should see "What tag set did you want to look at?"` here -- as you've probably noticed, we do a lot of checking error and success messages in our cukes, and for better or worse, we've always done it without checking that that it's in a flash div.